js/ui/select_box.js

```diff
@@ -510,7 +510,10 @@ const SelectBox = DropDownList.inherit({
             const initialSelectedItem = this.option('selectedItem');
 
             if(this.option('acceptCustomValue')) {
```

Comment:
```suggestion
            if(this.option('acceptCustomValue') && !saveEditingValue) {
                this._updateField(initialSelectedItem);
                this._clearFilter();
                return;
           }
```

Reply:
i think we should not change behavior for searchMode without acceptCustomValue in this case